client/src/sass/overrides.sass

```diff
@@ -77,6 +77,7 @@
 
 .v-tab.active-tab:hover, .v-tab--active
   background-color: white !important
+  pointer-events: none
```

Comment:
This is so the tab doesn't become clickable and go gray when you hover on it.

Reply:
Very cool